src/DotNet.Status.Web/TimelineIssueTriage.cs

```diff
@@ -22,7 +22,7 @@ public class TimelineIssueTriage : ITimelineIssueTriage
     {
         private static readonly string _docLink = "[Documentation](https://github.com/dotnet/arcade-services/blob/master/docs/BuildFailuresIssueTriage.md)";
         private static readonly string _markingLabelName = "darcbot";
-        private static readonly string[] _issueLabels = new[] { "Detected By - Customer", "First Responder", "Build Failed" };
+        private static readonly string[] _issueLabels = new[] { "Detected By - Ad Hoc", "First Responder", "Build Failed" };
```

Comment:
Does the label need to already exist for this code to work?  Worth ensuring it is created before usage.

Reply:
Yes and I changed to code to match the label I want to use. Thank you for the verification.